catalog/ui/catalog-ui-metacard-sharing-impl/src/main/java/org/codice/ddf/catalog/ui/sharing/ShareableMetacardImpl.java

```diff
@@ -116,8 +110,7 @@ public static ShareableMetacardImpl createOrThrow(Metacard metacard) {
     return Optional.of(new ShareableMetacardImpl(metacard));
   }
 
-  @VisibleForTesting
-  static ShareableMetacardImpl create(Map<String, Serializable> attributes) {
+  public static ShareableMetacardImpl create(Map<String, Serializable> attributes) {
```

Comment:
✏️  If we create instances using the factory method(s) they really should be next to the constructors and have javadoc on them.